tests/installation/start_install.pm

```diff
@@ -88,9 +88,10 @@ sub run {
         sleep 2;    # textmode is sometimes pressing alt-i too early
         send_key $cmd{install};
         wait_screen_change { send_key 'alt-o' } if match_has_tag('inst-overview-error-found', 0);
-        while (check_screen([qw(confirmlicense startinstall)], 5)) {
+        while (check_screen([qw(confirmlicense startinstall activate_flag_not_set)], 5)) {
             last if match_has_tag("startinstall");
-            send_key $cmd{acceptlicense};
+            send_key $cmd{acceptlicense} if match_has_tag("confirmlicense");
+            send_key 'alt-o';
```

Comment:
Now if needle matches 'confirmlicense' tag we will still press alt-o, and I'm not sure if that is required. Should it be pressed when match_has_tag activate_flag_not_set? As this tag is not used anywhere.

Reply:
Oups, patch updated to not press alt-o for license.
I also added above in first comment the ref to needle PR.